### DISCUSSION TITLE
DTU Command failed

### DISCUSSION THREAD

**Discussion by markusruebe** [tbnobody/OpenDTU#300] [Q&A]
Hi,
I started my openDTU some days ago and I like the project. In the meantime I found a project to automatically limit the Microconverter if there is more load produced that you need.

I now have DTU command failed messages in the Event Log. 
Is there a chance to investigate what errors that are in detail?
How could be checked what command failed to fix it.

Thanks for your hints and tricks.

**Comment by tbnobody** [comment]
Are you using a second device to set the limit or are you using the MQTT interface of OpenDTU?

**Comment by markusruebe** [comment]
Hi,
I use a raspberry with python script to get and set DTU data via MQTT. Data will be written to influx and grafana prepares visuals.

**Comment by tbnobody** [comment]
Please see https://github.com/tbnobody/OpenDTU/issues/230
You can talk to the inverter only using one device. Otherwise internal statuses get mixed up (e.g. in terms of the event log etc.)

Or did I understood you  wrong? Do you have a NRF24 module on the raspberry as well?
